spec.bs

```diff
@@ -335,18 +335,23 @@ A <dfn export>fenced frame config</dfn> is a struct with the following [=struct/
     "buyer",
     "seller",
     "component-seller",
+    "direct-seller",
     "shared-storage-select-url",
   };
 
   dictionary FenceEvent {
     required DOMString eventType;
     required DOMString eventData;
-    required sequence&lt;DOMString&gt; destination;
+    required sequence&lt;FenceReportingDestination&gt; destination;
   };
 
+  typedef (FenceEvent or DOMString) ReportEventType;
+
   [Exposed=Window]
   interface Fence {
-      undefined reportEvent(FenceEvent event);
+      undefined reportEvent(ReportEventType event);
+      undefined setReportEventDataForAutomaticBeacons(FenceEvent event);
+      sequence&lt;FencedFrameConfig&gt; getNestedConfigs();
```

Comment:
Pretty much looks good, but lets add method definitions for all of these with TODOs in them just so we don't forget to fill them out. You can CMD+F for "The <dfn method for="Navigation">entries()</dfn> method steps are:" in https://raw.githubusercontent.com/WICG/navigation-api/91c2e7f959418d71e26e5b9a6723fed2944001d9/spec.bs for an example.